pkg/providers/gcore/client.go

```diff
@@ -0,0 +1,86 @@
+package gcore
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+
+	gdns "github.com/G-Core/gcore-dns-sdk-go"
+	phonebook "github.com/pier-oliviert/phonebook/api/v1alpha1"
+)
+
+const (
+	EnvAPIURL   = "GCORE_API_URL"
```

Comment:
Does this need adding to the docs and samples or does it have a default set that I can't spot?

Reply:
Told you I was blind! 

Sounds like a plan to me 😃